src/libexpr/meson.build

```
@@ -47,11 +58,13 @@ if bdw_gc.found()
   ]
     define_name = 'HAVE_' + funcspec.underscorify().to_upper()
     define_value = cxx.has_function(funcspec).to_int()
-    configdata.set(define_name, define_value)
+    configdata_priv.set(define_name, define_value)
   endforeach
-  configdata.set('GC_THREADS', 1)
+  # Affects ABI, because it changes what bwg_gc itself does!
```

```suggestion
  # Affects ABI, because it changes what bdg_gc itself does!
```

---

```suggestion
  # Affects ABI, because it changes what bdw_gc itself does!
```

(maybe I'm missing a joke here, but I thought it was BDW as in Boehm-Demers-Weiser, heh)

---

hehe just typos. Thanks!!